software/openvisualizer/SConstruct

```diff
@@ -134,6 +136,12 @@ AddOption('--simTopology',
     type      = 'string')
 runnerEnv['SIMTOPOLOGY'] = GetOption('simTopology')
 
+AddOption('--importTopo',
```

Comment:
you use `importTopo` and `pathTopo`. I recommend you homogenize, if indeed these are the same thing.